### PR TITLE
fix(v8): explicitly publish dist folder after node 16 upgrade

### PR DESCRIPTION
**Pull request by TristanWatanabe**
## Changes
- Previous versions of v8 packages published the `dist/` folder prior to the node16 upgrade. This PR will now explicitly ship the `dist/` folder

## Related Issue(s)

- Follows #27745

### CODE REVIEW

**Comment by fabricteam** [comment]
## Perf Analysis (`@fluentui/react-northstar`)
<details><summary>Perf tests with no regressions</summary>

Scenario | Current PR Ticks | Baseline Ticks | Ratio
:--- | ---:| ---:| ---:
ReactionMinimalPerf.default | [277](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ReactionMinimalPerf.default.html) | [259](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ReactionMinimalPerf.default_base.html) | 1.07:1
LabelMinimalPerf.default | [286](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/LabelMinimalPerf.default.html) | [269](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/LabelMinimalPerf.default_base.html) | 1.06:1
SkeletonMinimalPerf.default | [256](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/SkeletonMinimalPerf.default.html) | [242](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/SkeletonMinimalPerf.default_base.html) | 1.06:1
TableMinimalPerf.default | [311](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/TableMinimalPerf.default.html) | [293](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/TableMinimalPerf.default_base.html) | 1.06:1
AlertMinimalPerf.default | [197](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/AlertMinimalPerf.default.html) | [189](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/AlertMinimalPerf.default_base.html) | 1.04:1
AvatarMinimalPerf.default | [135](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/AvatarMinimalPerf.default.html) | [131](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/AvatarMinimalPerf.default_base.html) | 1.03:1
DropdownManyItemsPerf.default | [504](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/DropdownManyItemsPerf.default.html) | [488](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/DropdownManyItemsPerf.default_base.html) | 1.03:1
FlexMinimalPerf.default | [202](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/FlexMinimalPerf.default.html) | [197](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/FlexMinimalPerf.default_base.html) | 1.03:1
GridMinimalPerf.default | [247](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/GridMinimalPerf.default.html) | [239](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/GridMinimalPerf.default_base.html) | 1.03:1
MenuMinimalPerf.default | [640](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/MenuMinimalPerf.default.html) | [620](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/MenuMinimalPerf.default_base.html) | 1.03:1
IconMinimalPerf.default | [470](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/IconMinimalPerf.default.html) | [457](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/IconMinimalPerf.default_base.html) | 1.03:1
CarouselMinimalPerf.default | [349](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/CarouselMinimalPerf.default.html) | [341](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/CarouselMinimalPerf.default_base.html) | 1.02:1
DialogMinimalPerf.default | [584](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/DialogMinimalPerf.default.html) | [572](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/DialogMinimalPerf.default_base.html) | 1.02:1
LayoutMinimalPerf.default | [262](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/LayoutMinimalPerf.default.html) | [258](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/LayoutMinimalPerf.default_base.html) | 1.02:1
PopupMinimalPerf.default | [453](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/PopupMinimalPerf.default.html) | [444](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/PopupMinimalPerf.default_base.html) | 1.02:1
CustomToolbarPrototype.default | [1907](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/CustomToolbarPrototype.default.html) | [1871](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/CustomToolbarPrototype.default_base.html) | 1.02:1
TooltipMinimalPerf.default | [1685](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/TooltipMinimalPerf.default.html) | [1657](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/TooltipMinimalPerf.default_base.html) | 1.02:1
AnimationMinimalPerf.default | [372](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/AnimationMinimalPerf.default.html) | [369](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/AnimationMinimalPerf.default_base.html) | 1.01:1
AttachmentSlotsPerf.default | [841](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/AttachmentSlotsPerf.default.html) | [833](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/AttachmentSlotsPerf.default_base.html) | 1.01:1
ChatMinimalPerf.default | [533](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ChatMinimalPerf.default.html) | [528](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ChatMinimalPerf.default_base.html) | 1.01:1
CheckboxMinimalPerf.default | [1575](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/CheckboxMinimalPerf.default.html) | [1557](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/CheckboxMinimalPerf.default_base.html) | 1.01:1
DropdownMinimalPerf.default | [1839](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/DropdownMinimalPerf.default.html) | [1813](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/DropdownMinimalPerf.default_base.html) | 1.01:1
EmbedMinimalPerf.default | [2413](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/EmbedMinimalPerf.default.html) | [2388](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/EmbedMinimalPerf.default_base.html) | 1.01:1
InputMinimalPerf.default | [724](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/InputMinimalPerf.default.html) | [720](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/InputMinimalPerf.default_base.html) | 1.01:1
ItemLayoutMinimalPerf.default | [906](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ItemLayoutMinimalPerf.default.html) | [901](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ItemLayoutMinimalPerf.default_base.html) | 1.01:1
ListNestedPerf.default | [412](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ListNestedPerf.default.html) | [408](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ListNestedPerf.default_base.html) | 1.01:1
LoaderMinimalPerf.default | [242](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/LoaderMinimalPerf.default.html) | [240](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/LoaderMinimalPerf.default_base.html) | 1.01:1
MenuButtonMinimalPerf.default | [1271](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/MenuButtonMinimalPerf.default.html) | [1258](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/MenuButtonMinimalPerf.default_base.html) | 1.01:1
ProviderMergeThemesPerf.default | [967](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ProviderMergeThemesPerf.default.html) | [954](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ProviderMergeThemesPerf.default_base.html) | 1.01:1
ProviderMinimalPerf.default | [277](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ProviderMinimalPerf.default.html) | [275](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ProviderMinimalPerf.default_base.html) | 1.01:1
RadioGroupMinimalPerf.default | [333](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/RadioGroupMinimalPerf.default.html) | [330](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/RadioGroupMinimalPerf.default_base.html) | 1.01:1
AttachmentMinimalPerf.default | [100](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/AttachmentMinimalPerf.default.html) | [100](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/AttachmentMinimalPerf.default_base.html) | 1:1
BoxMinimalPerf.default | [252](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/BoxMinimalPerf.default.html) | [252](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/BoxMinimalPerf.default_base.html) | 1:1
ButtonOverridesMissPerf.default | [823](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ButtonOverridesMissPerf.default.html) | [823](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ButtonOverridesMissPerf.default_base.html) | 1:1
ButtonSlotsPerf.default | [410](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ButtonSlotsPerf.default.html) | [408](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ButtonSlotsPerf.default_base.html) | 1:1
DatepickerMinimalPerf.default | [4533](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/DatepickerMinimalPerf.default.html) | [4554](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/DatepickerMinimalPerf.default_base.html) | 1:1
DividerMinimalPerf.default | [261](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/DividerMinimalPerf.default.html) | [260](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/DividerMinimalPerf.default_base.html) | 1:1
FormMinimalPerf.default | [277](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/FormMinimalPerf.default.html) | [277](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/FormMinimalPerf.default_base.html) | 1:1
ListCommonPerf.default | [490](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ListCommonPerf.default.html) | [492](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ListCommonPerf.default_base.html) | 1:1
ListMinimalPerf.default | [382](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ListMinimalPerf.default.html) | [383](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ListMinimalPerf.default_base.html) | 1:1
PortalMinimalPerf.default | [116](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/PortalMinimalPerf.default.html) | [116](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/PortalMinimalPerf.default_base.html) | 1:1
SliderMinimalPerf.default | [906](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/SliderMinimalPerf.default.html) | [908](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/SliderMinimalPerf.default_base.html) | 1:1
SplitButtonMinimalPerf.default | [3042](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/SplitButtonMinimalPerf.default.html) | [3035](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/SplitButtonMinimalPerf.default_base.html) | 1:1
StatusMinimalPerf.default | [506](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/StatusMinimalPerf.default.html) | [507](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/StatusMinimalPerf.default_base.html) | 1:1
TableManyItemsPerf.default | [1434](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/TableManyItemsPerf.default.html) | [1441](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/TableManyItemsPerf.default_base.html) | 1:1
TextAreaMinimalPerf.default | [350](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/TextAreaMinimalPerf.default.html) | [349](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/TextAreaMinimalPerf.default_base.html) | 1:1
ToolbarMinimalPerf.default | [663](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ToolbarMinimalPerf.default.html) | [664](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ToolbarMinimalPerf.default_base.html) | 1:1
TreeWith60ListItems.default | [105](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/TreeWith60ListItems.default.html) | [105](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/TreeWith60ListItems.default_base.html) | 1:1
AccordionMinimalPerf.default | [104](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/AccordionMinimalPerf.default.html) | [105](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/AccordionMinimalPerf.default_base.html) | 0.99:1
ButtonMinimalPerf.default | [103](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ButtonMinimalPerf.default.html) | [104](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ButtonMinimalPerf.default_base.html) | 0.99:1
ChatDuplicateMessagesPerf.default | [187](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ChatDuplicateMessagesPerf.default.html) | [189](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ChatDuplicateMessagesPerf.default_base.html) | 0.99:1
ChatWithPopoverPerf.default | [242](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ChatWithPopoverPerf.default.html) | [245](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ChatWithPopoverPerf.default_base.html) | 0.99:1
HeaderMinimalPerf.default | [260](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/HeaderMinimalPerf.default.html) | [263](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/HeaderMinimalPerf.default_base.html) | 0.99:1
HeaderSlotsPerf.default | [584](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/HeaderSlotsPerf.default.html) | [587](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/HeaderSlotsPerf.default_base.html) | 0.99:1
ImageMinimalPerf.default | [277](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ImageMinimalPerf.default.html) | [281](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ImageMinimalPerf.default_base.html) | 0.99:1
RefMinimalPerf.default | [142](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/RefMinimalPerf.default.html) | [143](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/RefMinimalPerf.default_base.html) | 0.99:1
ListWith60ListItems.default | [466](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ListWith60ListItems.default.html) | [476](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/ListWith60ListItems.default_base.html) | 0.98:1
RosterPerf.default | [1805](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/RosterPerf.default.html) | [1844](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/RosterPerf.default_base.html) | 0.98:1
SegmentMinimalPerf.default | [248](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/SegmentMinimalPerf.default.html) | [253](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/SegmentMinimalPerf.default_base.html) | 0.98:1
TreeMinimalPerf.default | [587](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/TreeMinimalPerf.default.html) | [596](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/TreeMinimalPerf.default_base.html) | 0.98:1
VideoMinimalPerf.default | [512](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/VideoMinimalPerf.default.html) | [524](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/VideoMinimalPerf.default_base.html) | 0.98:1
CardMinimalPerf.default | [375](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/CardMinimalPerf.default.html) | [388](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/CardMinimalPerf.default_base.html) | 0.97:1
TextMinimalPerf.default | [241](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/TextMinimalPerf.default.html) | [249](https://fluentuipr.z22.web.core.windows.net/pull/27769/perf-test-northstar/TextMinimalPerf.default_base.html) | 0.97:1

</details><div id="perfComment9424"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/048e93f6e918778673d196a6084ad2bfa872d5e7'>048e93f6e918778673d196a6084ad2bfa872d5e7</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 FluentUI-v0 No visual regressions between this PR and main
<div id="vrtCommentFluentUI-v0"/>

File: packages/common-styles/.npmignore [hunk 1]
```diff
@@ -29,3 +29,8 @@ tsd.json
 tslint.json
 typings
 visualtests
+
```
Comment (Hotell) [inline packages/common-styles/.npmignore:32]: this package will also ship `/src`

image

looking at some older version it seems to be expected https://unpkg.com/browse/@fluentui/common-styles@1.2.7/src/ 👀

**Review by Hotell — APPROVED** [review]
fyi: we have also migration generator for v8, would make sense to update it and use next time

https://github.com/microsoft/fluentui/tree/master/tools/generators/migrate-v8-pkg

**Comment by Hotell** [comment]
admin merged - approvals from v-build  + pipeline is passing

**Pull request merged**